### PR TITLE
✍️ Add Merchant Payment Conditions page

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Test plan:
- Expect to be able to navigate to https://docs.centrapay.com/guides/merchant-payment-conditions
- Expect to see Reference → Merchant Integrations → Merchant Payment Conditions in Navigation.

Screenshot 2023-01-23 at 3 37 54 PM

### CODE REVIEW

File: content/guides/merchant-payment-conditions.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Merchant Payment Conditions
+---
```
Comment (MeganSteenkamp) [inline content/guides/merchant-payment-conditions.md:3]: @tom-blake there is a comment that this guide highlight that it is a payment protocol extension.
Should we add a standard sentence to the top of the description?

File: content/guides/merchant-payment-conditions.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Examples of Merchant Payment Conditions include:
+
+- The merchant needs to confirm proof of identification for age-restricted purchases.
+- The merchant needs to confirm that a promotional item was purchased.
+- The patron needs to confirm the payment from their Centrapay integrated app.
+
+## Payment Conditions Flow
+
+In order to support Merchant Payment Conditions, the merchant integration must [create the Payment Request](https://docs.centrapay.com/api/payment-requests#create-a-payment-request) with the `conditionsEnabled` flag set to true.
```
Comment (tom-blake) [inline content/guides/merchant-payment-conditions.md:15]: @MeganSteenkamp To resolve your other comment, we could say "the merchant integration must extend Centrapay's payment protocol by creating the Payment Request..."

Comment (MeganSteenkamp) [reply content/guides/merchant-payment-conditions.md]: Thanks @tom-blake I have updated this in Notion and the PR.

File: components/content/ProseA.vue [hunk 3]
```diff
@@ -29,5 +29,5 @@ const props = defineProps({
 
 const isApiReferenceLink = props.href.startsWith('https://docs.centrapay.com/api');
 
-const isExternalLink = !props.href.startsWith('https://docs.centrapay.com') || isApiReferenceLink;
+const isExternalLink = (!props.href.startsWith('/') && !props.href.startsWith('https://docs.centrapay.com')) || isApiReferenceLink;
```
Comment (MeganSteenkamp) [inline components/content/ProseA.vue:32]: This allows for partial linking i.e. `/guides` for an internal link to run locally.

File: content/reference/merchant-integrations.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<!-- FIXME: Create a Merchant Integration landing page -->
+<meta http-equiv="refresh" content="0; URL=/guides/merchant-payment-conditions">
```
Comment (MeganSteenkamp) [inline content/reference/merchant-integrations.md:2]: This allows us to have the correct structure in our navigation and do a redirect if `Merchant Integrations` is clicked as we don't have a landing page.

Open to suggestions on how to do this, if we are planning to create a landing page I think this is ok as a placeholder.

Comment (tom-blake) [reply content/reference/merchant-integrations.md:2]: This is ok for now - there's legacy merchant terminals documentation that we should adapt for this page. I've added a task for this

**Review by tom-blake — APPROVED** [review]

**Pull request merged**